source/svgs/refills_logo.html.erb

```diff
@@ -0,0 +1,10 @@
+<svg fill="#93E9BE" role="img" aria-labelledby="refills-logo-title refills-logo-desc" xmlns="http://www.w3.org/2000/svg" width="612.4px" height="612.4px" viewBox="0 0 612.4 612.4">
```

Comment:
My thinking was that this is the logo's color, regardless of whether its on this site or any other. We're not restyling the color here, just displaying it as intended.